packages/app/src/features/liquidity-flow/hooks/use-borrow-first-position.tsx

```diff
@@ -79,7 +79,7 @@ export const useBorrowFirstPosition = (
     ),
   }
 
-  const modal = step ? modalByStep[step] : null
+  const modal = step ? modalByStep[step] || null : null
```

Comment:
```suggestion
  const modal = step && modalByStep[step] || null
```
But personally I'd prefer `modal?: JSX.Element` type and smth like `const modal = step && modalByStep[step]`

Reply:
fixed to const modal = step && modalByStep[step]